python/Ganga/new_tests/GPI/Parser/TestSelect.py

```diff
@@ -2,47 +2,68 @@
 
 from ..GangaUnitTest import GangaUnitTest
 
+job_num = 5
+job_names = ['a', 'b', 'c', 'd', 'e']
+
 class TestSelect(GangaUnitTest):
 
-    def test_SelectJob(self):
-        
-        from Ganga.GPI import Job, jobs
-        j=Job()
+    def setUp(self):
+        """Make sure that the Job object isn't destroyed between tests"""
+        super(TestSelect, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
```

Comment:
I'd like to avoid doing multi-pass style tests unless we really are testing the persistency features of Ganga. Tests really should be independent where possible unless there's a very good reason to couple them. We shouldn't be using tests functions in a class to group tests in order but rather to test separate things.